### PR TITLE
format TidalFlow2ACO in tutorials and create .rst 

### PR DESCRIPTION
**Pull request by hunjhunj**

### CODE REVIEW

File: docs/source/examples/TidalFlow2ACO/TidalFlow2ACO.rst [hunk 1]
```diff
@@ -0,0 +1,240 @@
+Sediment erosion and Transport from
+===================================
+
+In this notebook we implement the erosion and sediment transport
```
Comment (ale37911) [inline docs/source/examples/TidalFlow2ACO/TidalFlow2ACO.rst:4]: only erosion - no sediment transport. It assumes infinite amount of sediment in each cell, and only calcs. vertical erosion (we didn't have time to calc. the rest).

File: docs/source/examples/TidalFlow2ACO/TidalFlow2ACO.rst [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+should be run to build a new landlab installation containing the
+tidal-flow-calculator.
+
+We convert the matlab fuctions (totalsedimenterosionMUDsine.m and
+sedtran.m) from Mariotti and Murshid ( 2018) to python
+(tidal_erosion_calculator) to calculate total sediment erosion.
+
+.. code:: ipython3
+
+    from landlab.grid.mappers import map_link_vector_components_to_node
```
Comment (ale37911) [inline docs/source/examples/TidalFlow2ACO/TidalFlow2ACO.rst:35]: definitely importing more than just these two from landlab.grid.mappers..

**Review by ale37911 — COMMENTED** [review]
just making some comments on write up - thanks for starting this!

**Comment by elbeejay** [comment]
@ale37911 It is likely going to be easier for @hunjhunj to make your edits in her local notebook and re-building the `.rst` file so that the changes are reflected in both the `.ipynb` and on the rendered version. @ale37911 are you okay with merging as-is for now and doing another PR with corrections?

**Comment by ale37911** [comment]
Sure happy to do that - I'm playing with the notebook this morning a bit to see if I can track down error in computation of erosion...

**Pull request merged**